.circleci/config.yml

```diff
@@ -183,9 +183,8 @@ jobs:
 
   hardhat-core-default-solc:
     docker:
-      - image: circleci/node:16
+      - image: cimg/node:16.15
     steps:
-      - update-npm
```

Comment:
`cimg/node:16` comes with npm 8.11.0 so this is no longer needed. So we only need it for the one Truffle test that requires node.js 10 and I hope that we'll be able to update that one soon too.

Maybe we should replace this with a check for min npm version but I'm not sure if it's worth the hassle since this will become irrelevant soon.